core/transact.go

```diff
@@ -223,15 +222,15 @@ func (h *Handler) finalizeTxWait(ctx context.Context, txTemplate *txbuilder.Temp
 	return nil
 }
 
-func waitForTxInBlock(ctx context.Context, c *protocol.Chain, tx *bc.Tx, height uint64) (uint64, error) {
+func (h *Handler) waitForTxInBlock(ctx context.Context, tx *bc.Tx, height uint64) (uint64, error) {
```

Comment:
I actually needed to make it a method on `Handler` so that I could access `h.Submitter`. Or I could add the `Submitter` as another parameter to the function?